INSTALL.rst

```diff
@@ -257,6 +257,27 @@ location specified in ``haas.cfg``) and initialize its tables::
 
   haas init_db
 
+
+Authentication and Authorization
```

Comment:
I mentioned in a comment on another line, but configuring auth is mandatory, so relegating it to an extensions section seems unwise; it should be somewhere obvious.